src/core/plugins/oas3/selectors.js

```diff
@@ -61,6 +61,26 @@ export const shouldRetainRequestBodyValue = onlyOAS3((state, path, method) => {
   }
 )
 
+export const defaultRequestBodyValue = (state, path, method) => (system) => {
```

Comment:
should rename function to 'getDefaultRequestBodyValue'

Reply:
If I rename this function it will class with the already existing 'getDefaultRequestBodyValue' from "./components/request-body". May I also ask for the reason why this needs to have the 'get' prefix since the requestBodyValue function a few lines above it doesn't seem to have it either? Not sure what the naming conventions are.

Reply:
Good catch. Let's go with `selectDefaultRequestBodyValue` instead. Using `select<MethodName>` will be more inline with forward-looking naming conventions and reflective that it is a selector. As an example, using `select<MethodName>` is already used as the naming convention in swagger-editor@next.

Reply:
I've renamed it into `selectDefaultRequestBodyValue`. That should cover everything for this PR, let me know if anything else needs to be changed!